browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -1699,6 +1699,18 @@ Are you sure you want to do this?
       <message name="IDS_BRAVE_STATS_SHARE_TEXT" desc="Text sent when sharing the stats">
         Every day I save data by browsing the Web with Brave.
       </message>
+      <message name="IDS_WATCH_YOUTUBE_NOT_ADS" desc="Open youtube in brave modal title">
+        Watch YouTube, Not Ads.
```

Comment:
I believe `not` should not be capitalized here, but defer to @rmcfadden3.

Reply:
Yeah, @mkarolin , you're right. Should be `Watched YouTube, not ads.`

Reply:
Thank you @mkarolin @rmcfadden3 updating it in the commit